src/psyclone/tests/gocean/transformations/globalstoargs_test.py

```diff
@@ -0,0 +1,260 @@
+# -----------------------------------------------------------------------------
```

Comment:
I think it's my mistake but please could you put this file in `tests/domain/gocean/transformations`?

Reply:
Done